internal/indexmanagement/reconcile.go

```diff
@@ -336,13 +336,27 @@ func (imr *IndexManagementRequest) reconcileIndexManagementCronjob(policy apis.I
 		{Name: "POLICY_MAPPING", Value: mapping.Name},
 	}
 
-	if policy.Phases.Delete != nil {
+	if policy.Phases.Delete != nil { //#BEE: I think I need to read namespace value here
```

Comment:
This does not require to check if `policy.Phases.Delete.Namespaces` is set. If empty, the following code compiles an empty JSON list that can be checked in the python code. If empty, the python code can skip the delete by query routine.